### PR TITLE
[MSKINS-237] Rework skin for new site model

### PR DESCRIPTION
**Pull request by michael-o**

### CODE REVIEW

File: src/it/mskins-28/src/site/site.xml [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
+    <!-- Third and forth logo fit next to eachother in the sidebar -->
+    <logo name="Third Logo" alt="Third Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Fourth Logo" alt="Fourth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Fifth Logo" alt="Fifth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Sixth Logo" alt="Sixth Logo" img="./images/logos/maven-feather.png"/>
+    <logo name="Seventh Logo"/>
+  </poweredBy>
+  <!-- END SNIPPET: skin-custom-config -->
+
+</project>
```
Comment (hboutemy) [inline src/it/mskins-28/src/site/site.xml:58]: why don't we see the real diff? is there a CR/LF change?
I see 2 options:
- the CRLF change is accidental: let's avoid it
- the CRLF change is a fix to previous problem: let's do the fix in a separate commit

**Review by hboutemy — CHANGES_REQUESTED** [review]

File: src/it/mskins-28/src/site/site.xml (hunk 1 above)
Comment (michael-o) [reply src/it/mskins-28/src/site/site.xml:58]: This happens regularly because many files have been created before .gitattributes and never touched since. This is the outcome. So the change was not on purpose.

Comment (michael-o) [reply src/it/mskins-28/src/site/site.xml:58]: This affects only this single file, do you still want me to try to "fix" previos LS?

Comment (michael-o) [reply src/it/mskins-28/src/site/site.xml]: @hboutemy Done, have another look.

**Comment by michael-o** [comment]
@hboutemy Please have a look again.

**Review by hboutemy — APPROVED** [review]
LGTM, thanks for your hard work

**Comment by jira-importer** [comment]
Resolve #230
